swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, condition=0, fabric="Unknown"):
+        super().__init__(id)
+        self.condition = condition
```

Comment:
Using `__init__` from the parent class gives `Item` a chance to do any initialization it needs to in order to function properly.

For this project, Item's initialization sets the id **and** condition, therefore, we should pass `condition` to the parent function's constructor so that attribute is set on line 11 in `item.py`. Then we can remove line 6, which repeats what is already written in the parent class constructor.

```suggestion
        super().__init__(id, condition)
```